.gitignore

```diff
@@ -31,3 +31,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+
```

Comment:
Remove line no 34,35 and add Empty line at the end of file

Reply:
Done sir.